dimod/include/dimod/quadratic_model.h

```diff
@@ -612,6 +612,23 @@ class QuadraticModelBase {
         }
     }
 
+    /// Scale offset, linear biases, and interactions by a factor
+    void scale(double scale_factor) {
+        // adjust offset
+        offset() *= scale_factor;
+
+        // adjust linear biases and quadratic interactions
+        for (size_type ui = 0; ui < num_variables(); ui++) {
```

Comment:
This is a nit, but for consistency: we use `u` and `v` in the C++ code.

It's only in cython that we use `ui` and `vi` to distinguish the integer labels from the hashable labels.

Reply:
That is consistent with most of the file, although later in `change_vartype` I saw `ui` was used. Updated this.